app/src/main/java/de/madem/homium/databases/InventoryDao.kt

```diff
@@ -4,13 +4,16 @@ import androidx.room.Dao
 import androidx.room.Insert
 import androidx.room.Query
 import de.madem.homium.models.InventoryItem
-import java.util.*
+import kotlinx.coroutines.flow.Flow
 
 @Dao
 interface InventoryDao {
 
     @Query("SELECT * FROM inventoryItem")
-    fun fetchAllInventoryItems(): List<InventoryItem>
+    fun fetchAllInventoryItemsOld(): List<InventoryItem>
```

Comment:
Please mark this one with `@Deprecated` or a TODO to make sure that we don't forget the removal of this function

Reply:
Done